spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/ModuleDefinitionBuilder.java

```diff
@@ -83,8 +87,9 @@ else if (argument.getName().equalsIgnoreCase("outputType")) {
 				}
 			}
 			if (m > 0) {
-				builder.setParameter(BindingProperties.INPUT_BINDING_KEY,
-						String.format("%s.%d", streamName, m - 1));
+				builder.setParameter(BindingProperties.INPUT_BINDING_KEY, String.format("%s.%d", streamName, m - 1));
+				builder.setParameter(BindingProperties.INPUT_GROUP_KEY, DEFAULT_CONSUMER_GROUP_NAME);
+				builder.setParameter(BindingProperties.INPUT_DURABLE_SUBSCRIPTION_KEY,"true");
```

Comment:
Given this specific property is used only in rabbit binder and the consumer module is going to use the same `durable` queue that the upstream module would create, setting this property explicitly here may not be needed. I believe this is needed only for the named channel bindings where we **explicitly** create a `durable` queue for the consumer.